_config.yml

```diff
@@ -92,6 +93,14 @@ android-store: "Google Play"
 taet: "Telerik Application Enrollment Token"
 mvs: "Microsoft Visual Studio"
 
+# Aliases for the AB clients
+"ab-web": "in-browser client"
+"ab-win": "classic Windows desktop client"
+"ab-vse": "extension for Visual Studio"
+"ab-cli": "command-line interface"
+"ab-sp": "package for Sublime Text"
+"ab-proton": "universal desktop client"
+
```

Comment:
Not sure about these, too. First, with this spelling, they cannot be used in the beginning of a sentence or in a title. I suggest that we use the text replacement feature only for names of products, brands, etc.